site-builder/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "site-builder"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
Question: do we have any versioning strategy? 🤔

Reply:
I've been thinking of using something along those lines `MAJOR.MINOR.PATCH`.